lighthouse-core/test/runner-test.js

```diff
@@ -36,17 +38,23 @@ describe('Runner', () => {
     resetSpies();
   });
 
-  describe('Gather Mode & Audit Mode', () => {
+  describe.only('Gather Mode & Audit Mode', () => {
     const url = 'https://example.com';
     const generateConfig = _ => new Config({
       passes: [{
         gatherers: ['viewport-dimensions'],
       }],
       audits: ['content-width'],
     });
+    const artifactsPath = '.tmp/test_artifacts';
+    const resolvedPath = Runner._getArtifactsPath({auditMode: artifactsPath});
```

Comment:
since the expected behavior is that we support a path relative to `cwd`, I think we want an impl independent of the thing itself to test it (e.g. `const resolvedPath = path.resolve(process.cwd(), artifactsPath)` or whatever works from this file)